plugins/early_talker.js

```diff
@@ -18,11 +18,14 @@ exports.load_config = function () {
 
 exports.early_talker = function(next, connection) {
     var plugin = this;
-    if (!plugin.pause      ) { return next(); } // config set to 0
-    if (connection.relaying) { return next(); } // Don't pause AUTH/RELAY clients
+    if (!plugin.pause           ) { return next(); } // config set to 0
+    if (!connection.early_talker) { return next(); }
+    if ( connection.relaying    ) {   // Don't pause AUTH/RELAY clients
```

Comment:
If you do this; you'll got this logged for every relay client regardless as to whether or not they are earlytalkers or not.